docs/usage.md

```diff
@@ -57,7 +57,7 @@ An [example samplesheet](../assets/samplesheet.csv) has been provided with the p
 The typical command for running the pipeline is as follows:
 
 ```console
-nextflow run nf-core/metapep --input samplesheet.csv --genome GRCh37 -profile docker
+nextflow run nf-core/metapep --input samplesheet.csv  --outdir <OUTDIR> --genome GRCh37 -profile docker
```

Comment:
This document isnt updated yet, is it? Because `--genome GRCh37` seems not correct?

Reply:
The document is updated, but I forgot to delete --genome here. I will fix it in the PR for the documentation.

Reply:
No, it's not yet updated, I will do that in a later PR.